src/lib/components/Grid.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+interface Gap {
+  row: number;
+  column: number;
+}
```

Comment:
```suggestion
type Gap = {
  row: number;
  column: number;
} | number;
```
Gap의 축약 표현형으로 `number`를 허용하는 것을 생각해본다면 `Gap` 타입에는 `number` 도 포함되어야 하지 않을까요? 혹시 어떻게 생각하시나용

Reply:
오호! 이렇게 타입을 정의하는 것이 더 맞겠네요~! 👍